Content.Shared/Hands/Components/SharedHandsComponent.cs

```diff
@@ -413,11 +429,16 @@ private void RemoveHeldEntityFromHand(Hand hand)
             if (hand.Name == ActiveHand)
                 DeselectActiveHeldEntity();
 
+            Owner.EntityManager.EventBus.RaiseLocalEvent(heldEntity.Uid, new RemovedFromHandEvent(Owner), false);
+
             if (!handContainer.Remove(heldEntity))
             {
                 Logger.Error($"{nameof(SharedHandsComponent)} on {Owner} could not remove {heldEntity} from {handContainer}.");
                 return;
             }
+
+
+
```

Comment:
space

Reply:
commenting this here but apparently an unequippedhandevent already existed so oops removed this entirely